_dockerPublisher/Dockerfile

```diff
@@ -3,7 +3,7 @@
 ############################################################################
 
 ####### BASE #################
-FROM fedora:31
+FROM fedora:34
```

Comment:
the latest is fedora 35. Don't you want to upgrade to the latest as it has longer time to be abandoned.